### DISCUSSION TITLE
How can I achieve to this result

### DISCUSSION THREAD

**Discussion by pierrefaidherbe** [prisma/prisma#13209] [Q&A]
I have this Schema:

one A has many B

one B has many C

How can I have all C for each A ?

in Prisma way or at least in raw sql (I've a little big forgot my sql...)

thanks you!

**Comment by nurul3101** [comment]
Hey @pierrefaidherbe 👋 

Here's how you could model A has one to many relations to B and B has one to many relations to C.
Can you please elaborate on all C for each A point?

```prisma
model A {
  id String @id @default(cuid())
  b  B[]
}

model B {
  id   String @id @default(cuid())
  a    A      @relation(fields: [a_id], references: [id])
  a_id String
  c    C[]
}

model C {
  id   String @id @default(cuid())
  b    B      @relation(fields: [b_id], references: [id])
  b_id String
}
```

**Reply by pierrefaidherbe** [reply]
Hi @nurul3101 ! Thank you for your response ! 

I would have like to have a that kind of response

`[A:[C:{...}, C:{...}], A:[C:{...}, C:{...}]]`

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
